Makefile

```diff
@@ -1,16 +1,15 @@
 run:
 	bash -ex bin/job /var/tmp/dst disk ~/tmp/test-src/disk.raw
 
-container-build: images/v2v-job/Dockerfile images/v2v-pv-populator/Dockerfile
-	docker build -t quay.io/fabiand/v2v-job -f images/v2v-job/Dockerfile .
-	docker build -t quay.io/fabiand/v2v-pv-populator -f images/v2v-pv-populator/Dockerfile .
+container-build: images/v2v-job/Dockerfile
+	docker build -t quay.io/pkliczewski/v2v-job -f images/v2v-job/Dockerfile .
```

Comment:
yuo might want to use a variable for the org:

```
docker build -t quay.io/pkliczewski/v2v-job -f images/v2v-job/Dockerfile .
CHANGE TO
QUAY_ORG=kubevirt
docker build -t quay.io/$QUAY_ORG/v2v-job -f images/v2v-job/Dockerfile .
```

Then you can override it at build time `make QUAY_ORG=pkliczewski …`

Reply:
ack